Add SEO (#83)
OpenLiberty/guide-microprofile-opentracing: README.adoc

:page-permalink: /guides/{projectid}
:page-related-guides: ['cdi-intro']
:common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master

> :page-seo-title: Enabling distributed tracing in microservices using Eclipse MicroProfile OpenTracing

Reviewer: ```suggestion
:page-seo-title: Enabling distributed tracing in Java microservices using Eclipse MicroProfile OpenTracing
```

> :page-seo-description: A tutorial on how to enable distributed tracing in Java microservices to easily trace the flow of a request that spans multiple resources by using MicroProfile OpenTracing and a Zipkin tracing system.

Reviewer: ```suggestion
:page-seo-description: A getting started tutorial and an example on how to enable distributed tracing in Java microservices to easily trace request flows that span multiple resources by using MicroProfile OpenTracing and Zipkin tracing system.
```
